recipes/tethys-platform/meta.yaml

```diff
@@ -0,0 +1,119 @@
+{% set name = "tethys-platform" %}
+{% set version = "3.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tethysplatform/tethys/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 3775d5599e756206204e09ca9a2114c8866fad7fd29999b256343c847ada3b1f
+
+build:
+  noarch: python
+  number: 0
+  ignore_run_exports:
+    - pip
+    - pbr
```

Comment:
```suggestion
```
I believe this is unrelated to failed Windows builds from earlier. Regardless, it shouldn't be necessary.